test/tests/functional/pbs_mom_dynamic_resource.py

```diff
@@ -293,57 +277,67 @@ def test_t10(self):
         self.server.expect(JOB, {'job_state': 'Q', 'comment': c},
                            id=jid, attrop=PTL_AND)
 
-    def test_t11(self):
+    def test_delete_res(self):
         """
         delete a resource while jobs are running/queued.
         """
-        resc_name = "foo"
-        self.create_mom_resources(resc_name, "string", "h", "/bin/echo abc")
 
-        a = {'scheduling': 'True'}
-        self.server.manager(MGR_CMD_SET, SERVER, a)
+        resc_name = ["foo"]
+        resc_type = ["string"]
+        resc_flag = ["h"]
+        script_body = ["/bin/echo abc"]
+
+        self.create_mom_resources(resc_name, resc_type, resc_flag, script_body)
 
         # Submit a job that requests mom dynamic resource
-        attr = {"Resource_List." + resc_name: 'abc'}
+        attr = {"Resource_List." + resc_name[0]: 'abc'}
         j = Job(TEST_USER, attrs=attr)
         jid = self.server.submit(j)
 
         # The job should run
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
 
+        # Check for the expected error message throwing by PBS
         try:
             self.server.manager(MGR_CMD_DELETE, RSC, id='foo')
         except PbsManagerError as e:
-            self.logger.info(e.message)
+            self.logger.info("As expected got error message: " + e.message)
```

Comment:
you are not checking for anything but just logging, update this to assertTrue (message in e.message)